src/goto-programs/goto_trace.h

```diff
@@ -86,6 +86,8 @@ class goto_trace_stept
   enum class assignment_typet { STATE, ACTUAL_PARAMETER };
   assignment_typet assignment_type;
 
+  // the instruction that created this step
```

Comment:
Clarify that calls are outside a function and returns inside it?

Reply:
ok